github/teams.go

```diff
@@ -355,3 +355,103 @@ func (s *TeamsService) ListUserTeams(ctx context.Context, opt *ListOptions) ([]*
 
 	return teams, resp, nil
 }
+
+// ListTeamProjects lists the organization projects for a team.
+//
+// GitHub API docs: https://developer.github.com/v3/teams/#list-team-projects
+func (s *TeamsService) ListTeamProjects(ctx context.Context, id int64) ([]*Project, *Response, error) {
+	u := fmt.Sprintf("teams/%v/projects", id)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	acceptHeaders := []string{mediaTypeNestedTeamsPreview, mediaTypeProjectsPreview}
+	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
+
+	var projects []*Project
+	resp, err := s.client.Do(ctx, req, &projects)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return projects, resp, nil
+}
+
+// ReviewTeamProjects checks whether a team has read, write, or admin
+// permissions for an organization project.
+//
+// Github API docs: https://developer.github.com/v3/teams/#review-a-team-project
+func (s *TeamsService) ReviewTeamProjects(ctx context.Context, id, projectID int64) (*Project, *Response, error) {
```

Comment:
Nit: I would have gone with `teamID` instead of `id` for methods required multiple ID's.

Reply:
Good point, @gauntface ... I agree. Let's please change this @palash25.

Reply:
Done :+1: